src/content/en/2021/privacy.md

```diff
@@ -5,572 +5,541 @@ description: Privacy chapter of the 2021 Web Almanac covering adoption and impac
   sheets_gid="1662197018",
   sql_file="nb_websites_with_iab.sql"
   )
 }}
 
-In the 10 most popular consent management platforms that are part of the framework, at the top we find <a hreflang="en" href="https://www.quantcast.com/products/choice-consent-management-platform/">Quantcast</a> with 0.34% on mobile and 0.32% on desktop. Other popular solutions are <a hreflang="en" href="https://www.didomi.io/">Didomi</a> and Wikia, with 0.3% and 0.23% on mobile and desktop respectively.
-
-
+Above, we show the distribution of the usage of both versions of TCF and of USP. Note that the crawl is US-based, therefore we do not expect many websites to have implemented TCF. Fewer than 2% of websites use any TCF version, while twice as many websites use the US Privacy framework.
 
 {{ figure_markup(
   image="most_common_cmps_for_iab_tcf_v2.png",
   caption="10 most popular consent management platforms for IAB ",
-  description="Bar chart showing the 10 most popular consent management platforms used for IAB's compliance frameworks.",
+  description="Bar chart showing the 10 most popular consent management platforms used for IAB's compliance frameworks. Quantcast is used on 0.31 of desktop sites and 0.33% of mobile sites, Didomi on 0.29% and 0.24% respectively, Wikia, Inc. on 0.23% and 0.19%, Google LLC on 0.08% and 0.09%, SIRDATA on 0.06% and 0.08%, iubenda on 0.07% and 0.07%, OneTrust LLC on 0.05% and 0.06%, Sourcepoint on 0.05% and 0.05%, consentmanager.net on 0.03% and 0.02%, and finally LiveRamp on 0.02% and 0.01%.",
   chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vRfOwcNkLp-mYBkmhDV5AFxl8p0ls9oxFnmmo0WUcAJxjxQqmHjquRZCWj_lNZRyFtX3RdH5T92IESu/pubchart?oid=2085725571&format=interactive",
   sheets_gid="692122193",
   sql_file="most_common_cmps_for_iab_tcf_v2.sql"
   )
 }}
 
+In the 10 most popular consent management platforms that are part of the framework, at the top we find <a hreflang="en" href="https://www.quantcast.com/products/choice-consent-management-platform/">Quantcast</a> with 0.34% on mobile. Other popular solutions are <a hreflang="en" href="https://www.didomi.io/">Didomi</a> with 0.24%, and Wikia, with 0.30%.
 
-In the USP framework, the website's and user's privacy settings are encoded in a "<a hreflang="en" href="https://github.com/InteractiveAdvertisingBureau/USPrivacy/blob/master/CCPA/US%20Privacy%20String.md">privacy string</a>". The most common privacy string is `1---`. This indicates that CCPA does not apply to the website and therefore the website not obliged to provide an opt-out for the user. CCPA only applies to companies whose main business involves selling personal data, or to companies that process data and have an annual turnover of more than $25 million. The second most recurring string is `1YNY`. This indicates that the website provided "notice and opportunity to opt-out of sale of data", but that the user has _not_ opted out of the sale of their personal data.
+{# TODO why no link for Wikia? #}
```

Comment:
Question

Reply:
Wikia appears to have a custom CMP, but doesn't provide it to other websites. It just happens to have a lot of sites in CRuX probably, which makes it one of the biggest "CMP providers". I didn't add a link because there's basically nothing to see there (about CMPs).

Reply:
Just feels a bit weird that it's the only one not linked. But if don't have a relevant link then not force on in for the sake of it. Resolving.